sycl/include/sycl/ext/oneapi/experimental/graph.hpp

```diff
@@ -340,8 +349,10 @@ class __SYCL_EXPORT executable_command_graph {
   /// Constructor used by internal runtime.
   /// @param Graph Detail implementation class to construct with.
   /// @param Ctx Context to use for graph.
+  /// @param EnableProfiling Enable graph profiling.
   executable_command_graph(const std::shared_ptr<detail::graph_impl> &Graph,
-                           const sycl::context &Ctx);
+                           const sycl::context &Ctx,
+                           const bool EnableProfiling = false);
```

Comment:
In https://github.com/intel/llvm/pull/12840/ a similar change was made (adding a property on finalize) but the approach used was to forward the property list to this constructor. I have a preference for that approach (since it avoids the need to add new parameters every time for properties here) so unless you think this is better I would prefer if they were aligned.

Reply:
The code has been updated this way.